namespaces/live.cloud-platform.service.justice.gov.uk/offender-management-production/resources/rds.tf

```diff
@@ -17,7 +17,7 @@ module "allocation-rds" {
   environment-name       = "production"
   infrastructure-support = "manage-pom-cases@digital.justice.gov.uk"
   db_engine              = "postgres"
-  db_engine_version      = "14.3"
+  db_engine_version      = "14.7"
```

Comment:
I would recommend explicitly setting this var to false

https://github.com/ministryofjustice/cloud-platform-terraform-rds-instance#input_allow_minor_version_upgrade

OR

if you are happy with minor upgrades then specify:

`db_engine_version = 14`

Reply:
Ah, that's what went wrong.... OK will do that.